README.md

```diff
@@ -33,6 +33,40 @@ The deployment is completed when HCO custom resource reports its condition as `A
 
 For more explanation and advanced options for HCO deployment using kustomize, refer to [kustomize deployment documentation](deploy/kustomize/README.md).
 
+##
+
+## Building Custom Bundle and Index Image
+As prerequisites, [OLM](https://sdk.operatorframework.io/docs/installation/) is required.
+If you wish to use cert-manager for certificates, the script in hack/deploy-cert-manager.sh might be helpful.
+
+To build a custom bundle and index image, with [opm](https://docs.redhat.com/en/documentation/openshift_container_platform/4.9/html/cli_tools/opm-cli) installed run:
+```bash
+# Set your image repository
+export REGISTRY_NAMESPACE=<your-org>
+export IMG_TAG=<your-tag>
+
+export HCO_OPERATOR_IMAGE="quay.io/$REGISTRY_NAMESPACE/hyperconverged-cluster-operator:$IMG_TAG"
+export HCO_WEBHOOK_IMAGE="quay.io/$REGISTRY_NAMESPACE/hyperconverged-cluster-webhook:$IMG_TAG"
+export HCO_DOWNLOADS_IMAGE="quay.io/$REGISTRY_NAMESPACE/virt-artifacts-server:$IMG_TAG"
```

Comment:
Nothing here builds the HCO images. Also, Makefile and the scripts uses `IMAGE_TAG`, rather than `IMG_TAG`. I would change it to:

```suggestion
export IMAGE_TAG=<your-tag>

make build-push-multi-arch-operator-image build-push-multi-arch-webhook-image build-push-multi-arch-artifacts-server

export HCO_OPERATOR_IMAGE="quay.io/${REGISTRY_NAMESPACE}/hyperconverged-cluster-operator:${IMAGE_TAG}"
export HCO_WEBHOOK_IMAGE="quay.io/${REGISTRY_NAMESPACE}/hyperconverged-cluster-webhook:${IMAGE_TAG}"
export HCO_DOWNLOADS_IMAGE="quay.io/${REGISTRY_NAMESPACE}/virt-artifacts-server:${IMAGE_TAG}"
```